python_modules/dagster/dagster/_core/definitions/definitions_class.py

```diff
@@ -29,7 +29,6 @@
     from dagster._core.storage.asset_value_loader import AssetValueLoader
 
 
-@experimental
```

Comment:
Not critical for this PR, but a few comments on the docstring that I noticed while reviewing:
- "Create a set of definitions explicitly available and loadable by Dagster tools." - now that this is a class instead of a method, maybe take out the "Create". Also, this should probably be the first line?
- The bullet points blow past line width.
- It's not clear to me what's meant by "do not apply to vanilla Dagster definitions".

Reply:
The bullets need to be on a single line for them to be appropriately parsed by sphinx. Or at least that was the only way I was able to get sphinx compile.

Reply:
Sandy how about "Definitions provides a few conveniences for dealing with resources that do not apply to definitions groups with ``@repository` declarations."

Reply:
@sryza  we can discuss in https://github.com/dagster-io/dagster/pull/11134